### PR TITLE
Remove MergeConfig() and SetOutputAXI()

### PR DESCRIPTION
**Pull request by njhollinghurst**
Also an unrelated commit that I think fixes a bug in backend_debug?

### CODE REVIEW

File: src/libpisp/frontend/frontend.hpp [hunk 1]
```diff
@@ -53,8 +53,6 @@ class FrontEnd
 	void SetOutputFormat(unsigned int output_num, pisp_image_format_config const &output_format);
 	void SetOutputBuffer(unsigned int output_num, pisp_fe_output_buffer_config const &output_buffer);
 	void SetOutputIntrLines(unsigned int output_num, int lines);
-	void SetOutputAXI(pisp_fe_output_axi_config const &output_axi);
```
Comment (naushir) [inline src/libpisp/frontend/frontend.hpp:56]: Do we definitely not want user control of the AXI params?

**Comment by naushir** [comment]
Looks good, just a minor question re the AXI output removal.

**Comment by njhollinghurst** [comment]
Actually I think we can remove HOG as well now. 
It is needed by 2712 chip tests, but for those we can specify to use the tagged version of libisp v1.0.2.

File: src/libpisp/frontend/frontend.hpp (hunk 1 above)
Comment (naushir) [reply src/libpisp/frontend/frontend.hpp:56]: Sounds reasonable.

**Comment by njhollinghurst** [comment]
Oh, too late to remove HOG.  Worth doing as a separate commit?

**Pull request merged**